content/millennium/dstu2.md

```diff
@@ -533,6 +533,7 @@ The possible `relation` values are:
 --------|------------------------------------------------------
  `self` | Shows the URL of the current page of results.
  `next` | Shows the URL of the immediate next page of results.
```

Comment:
nit: can we keep the table columns aligned in the md?

Reply:
https://github.com/cerner/fhir.cerner.com/pull/381/commits/97145ae812d544d0be308122b426766d2b00c5cd